charts/website/Chart.yaml

```
@@ -10,12 +10,11 @@ sources:
   - https://github.com/waldo-vision/waldo/tree/master/apps/web
 
 # chart version, should follow semver
-version: 0.30.0
+version: 0.32.0
```

What happened to version 31?
```suggestion
version: 0.31.0
```